examples/pl_loops/yielding_training_step.py

```diff
@@ -64,7 +64,7 @@ def connect(self, **kwargs):
     def on_run_start(self, optimizers, kwargs):
         super().on_run_start(optimizers, kwargs)
         if not inspect.isgeneratorfunction(self.trainer.lightning_module.training_step):
-            raise MisconfigurationException("The `LightningModule` does not yield anything in the `training_step`.")
+            raise _RuntimeError("The `LightningModule` does not yield anything in the `training_step`.")
```

Comment:
This is an exampple so you can just use `RuntimeError`

Reply:
:thumbsup: